addons/spectator/XEH_postInit.sqf

```diff
@@ -12,3 +12,15 @@ if (isServer) then {
 };
 
 [QGVAR(stageSpectator), FUNC(stageSpectator)] call CBA_fnc_addEventHandler;
+
+// Initalise virtual spectators
+[QGVAR(virtual),"initpost",{
+    if !(GVAR(isSet)) then {
+        // Delay until local player is fully initalized
+        [
+            { !isNil { player } && { !isNull player } },
```

Comment:
No it should always be `player`. Good question though, I'll add a comment that says this explicitly